helm/higress/values.yaml

```
@@ -603,3 +603,10 @@ pilot:
 
   # Additional labels to apply on the pod level for monitoring and logging configuration.
   podLabels: {}
+
+
+# Skywalking config settings
+enableSkywalking: false
+Skywalking:
+  address: "skywalking-oap.istio-system.svc"  # Must be set explicitly
```

```suggestion
  address: "skywalking-oap.higress-system.svc"  # Must be set explicitly
```
or skywalking-system ?